src/pages/Admin/WorldUsers/WorldUsers.tsx

```diff
@@ -55,7 +55,9 @@ export const WorldUsers: React.FC = () => {
 
   const owners = world?.owners;
 
-  const { spaces } = useWorldSpaces({ worldId: worldId ?? ("" as WorldId) });
+  const { spaces: worldSpaces } = useWorldSpaces({
+    worldId: worldId ?? ("" as WorldId),
```

Comment:
Maybe `undefined` should be legal input and the hook inside utilize `DEFERRED` to stave off query until the parameter is available.

I'd also think a `@debt` comment explaining the above inside the code would be enough for this PR